cross-chain/arbitrum/external/goerliEthereum/TokenBridge.json

```diff
@@ -0,0 +1,74 @@
+{
+  "address": "0xF890982f9310df57d00f659cf4fd87e65adEd8d7",
```

Comment:
Same comment as for Goerli Arbitrum `TokenBridge`: I can see [the address is correct](https://book.wormhole.com/reference/contracts.html) but I can't verify the ABI. Where does it come from? Do we need it or are we fine with just the address?

Reply:
ABI was copied from Etherscan. But since we don't use it, it's okay to leave the address only. I removed the rest.